src/Microsoft.TemplateEngine.Utils/TemplateInfoExtensions.cs

```diff
@@ -0,0 +1,42 @@
+using Microsoft.TemplateEngine.Abstractions;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.TemplateEngine.Utils
+{
+    /// <summary>
+    /// The class provides ITemplateInfo extension methods
+    /// </summary>
+    public static class TemplateInfoExtensions
+    {
+        /// <summary>
+        /// Gets the language defined in <paramref name="template"/>
+        /// </summary>
+        /// <param name="template">template definition</param>
+        /// <returns>The language defined in the template or null if no language is defined</returns>
+        /// <remarks>The tags are read in <see cref="SimpleConfigModel.ConvertedDeprecatedTagsToParameterSymbols"/> method. The single value for the tag is guaranteed.</remarks>
+        public static string GetLanguage(this ITemplateInfo template)
+        {
+            if (template.Tags == null || !template.Tags.TryGetValue("language", out ICacheTag languageTag))
```

Comment:
nit: The only difference between this method and `GetTemplateType` is the string literal on this line. Would it make sense to eliminate a bit of code duplication and refactor the two to use a common helper?